example/lib/peripheral_details/components/peripheral_details_view.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+class PeripheralDetailsView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Text('Details view');
```

Comment:
IMHO wrapping a blank text, without any formatting, etc. is not necessary

Reply:
Refactored this to a single Text in the parent view.